.github/ISSUE_TEMPLATE/support---question-and-answer.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Support
+about: "Learn how to interact with the PGO community"
+---
+
+If you believe you have found have found a bug, please open up [Bug Report](https://github.com/CrunchyData/postgres-operator/issues/new?template=bug_report.md)
+
+If you have a feature request, please open up a [Feature Request](https://github.com/CrunchyData/postgres-operator/issues/new?template=feature_request.md)
+
+You can find information about general PGO [support](https://access.crunchydata.com/documentation/postgres-operator/latest/support/) at:
+
+[https://access.crunchydata.com/documentation/postgres-operator/latest/support/](https://access.crunchydata.com/documentation/postgres-operator/latest/support/)
```

Comment:
we need to rework this file.
Like remove crunchy links, change github issues links to our jira
@spron-in

Reply:
we should remove issue_template at-all